src/excmds.ts

```diff
@@ -1821,7 +1821,7 @@ export async function containercreate(name: string, color?: string, icon?: strin
   @param name The container name.
  */
 //#background
-export async function containerremove(name: string) {
+export async function containerdelete(name: string) {
```

Comment:
Secret breaking changes are double-plus-ungood :(

Please leave an alias for the old command.

Reply:
Duly noted. (mainly changed it because of the typo in the changelog though ;))